File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/ServerOrderFieldLabel.scala

package org.openapitools.client.model


case class ServerOrderFieldLabel (
    /* Name of the field label. */
    _name: Option[String],
    /* Active status of the field label. */
    _active: Option[Integer]
)
object ServerOrderFieldLabel {
    def toStringBody(var_name: Object, var_active: Object) =
        s"""
        | {
        | "name":$var_name,"active":$var_active
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/zapier/models/ServerOrderFieldLabel.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}name`,
                label: `Name of the field label. - [${labelPrefix}name]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}active`,
                label: `Active status of the field label. - [${labelPrefix}active]`,
                type: 'integer',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'name': bundle.inputData?.[`${keyPrefix}name`],
            'active': bundle.inputData?.[`${keyPrefix}active`],
        }
    },
}
